source/stylesheets/components/_highlight.scss

```diff
@@ -0,0 +1,161 @@
+.highlight {
+  border-radius: $base-border-radius;
+  font-family: Menlo, monospace;
+  font-size: 0.8em;
+  line-height: 1.5em;
+  margin: $base-spacing 0 $large-spacing;
+  overflow: hidden;
+  position: relative;
+
+
+  &.handlebars {
+    .inline,
+    .inline-delimiter,
+    .delimiter {
+      color: lighten(green, 10%);
+    }
+
+    .inline {
+      .attribute-name {
+        color: green;
+      }
+
+      .attribute-value,
+      .content {
+        color: darken(green, 10%);
+      }
+    }
+  }
+
+  .scroller {
+    overflow: auto;
+  }
+
+  pre {
+    margin: 0;
+  }
+
+  table {
+    background-color: $code-background;
+    border: none;
+    margin: 0;
+    table-layout: auto;
+    width: 100%;
+
+    tr,
+    th,
+    td {
+      border: none;
+    }
+
+    th,
+    td {
+      padding: 5px 10px;
+    }
+
+    td {
+      color: $white;
+
+      &.line-numbers {
+        background-color: $code-line-number-background;
+        color: $medium-gray;
+        text-align: center;
+        width: 2em;
+      }
+
+      &.code {
+        padding: 0.5em 0 0.5em 1em;
+      }
+    }
+
+    thead {
+      background-color: $code-header-background;
+
+      td {
+        color: $ember-orange;
+      }
+    }
+
+    &.specific {
+      th, td {
+        padding: 5px 6px;
+      }
+    }
+
+    &.extensive {
+      font-size: 9px;
+    }
+
+    /* HTML */
+
+    @each $tag, $color in $highlight-colors {
+      .#{$tag} {
+        color: $color;
+      }
+    }
+  }
+
+  .ribbon {
+    @include position(absolute, null 0 6px null);
+    @include size(52px 20px);
+  }
+
+  &.javascript .ribbon,
+  &.js .ribbon {
+    //@include hidpi('js-ribbon', png);
+  }
+  &.html .ribbon {
+    //@include hidpi('html-ribbon', png);
+  }
+  &.handlebars .ribbon {
+    //@include hidpi('handlebars-ribbon', png);
+  }
+
+  .highlight-line {
+    background-color: $highlight-yellow;
+    border-left: $highlight-yellow solid 5px;
+    border-right: $highlight-yellow solid 5px;
+    box-sizing: content-box;
+    display: inline-block;
+    margin: 0 -10px;
+
+    &.added {
+      border-color: $highlight-green;
+      background-color: $highlight-green;
+    }
+
+    &.removed {
+      border-color: $highlight-red;
+      background-color: $highlight-red;
+    }
+  }
+
+  .code .highlight-line {
+    border-left-width: 13px;
+    border-right-width: 13px;
+    margin: 0 -13px;
+    width: 613px;
+  }
+
+  .line-numbers .highlight-line {
+    width: 28px;
+    border-right-color: darken($highlight-yellow, 20%);
+
+    &.added {
+      border-right-color: darken($highlight-green, 20%);
+    }
+
+    &.removed {
+      border-right-color: darken($highlight-red, 20%);
+    }
+  }
+}
+
+code {
+  background-color: $base-border-color;
```

Comment:
No it doesn't, it's for the inline code (like `Ember.ArrayController` in the screenshot), and `.highlight` just goes on the big code blocks. This could go somewhere else, but I figure they're semi-related.